src/extension.ts

```diff
@@ -134,14 +132,13 @@ async function backgroundInit(context: vscode.ExtensionContext) {
   setDefaultStatus();
   void registerCommands(context);
   pollDownloadProgress();
+  pollProcessState(() => {
+    void handleOpenWelcomeInHub(context);
+  });
```

Comment:
won't this open the welcome/hub to all users? how will we implement the AB test? is it the `should_open_welcome_in_hub` flag?

Reply:
No, the logic for opening the hub is in `should_open_welcome_in_hub` and yes it is using the flag from the binary